util/src/main/java/io/camunda/zeebe/util/buffer/BufferUtil.java

```diff
@@ -114,6 +114,14 @@ public static DirectBuffer cloneBuffer(
     }
   }
 
+  public static DirectBuffer trimBufferIfExceeds(final DirectBuffer src, final int maxLength) {
```

Comment:
🔧 Usually when we a String is cut off this is indicated by appending `...` at the end. I feel like we should do this for these messages as well so it's clear to users that this is not the entire message.

Reply:
Really good one! This suggestion lead me to use existing method `limitString` which already truncates the string and puts `...` in the end. Therefore, I do not need to create a new method in `BufferUtil`.